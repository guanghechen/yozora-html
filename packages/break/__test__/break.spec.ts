import type { IBreak } from '@yozora/ast'
import renderBreak from '../src'

describe('snapshot', function () {
  it('basic', function () {
    const node: IBreak = {
      type: 'break',
    }
    expect(renderBreak(node)).toMatchSnapshot()
  })
})
