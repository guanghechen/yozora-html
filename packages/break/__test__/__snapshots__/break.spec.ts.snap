// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot basic 1`] = `"<br class="yozora-break" />"`;
